packages/shared/src/storage/secureStorage/index.ts

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+export const setSecureItem = async (key: string, data: string) =>
```

Comment:
声明共享的 types，三个 index 使用 同一个 type 导出

```suggestion
import { ISecureStorage } from './types'

const storage: ISecureStorage = {
    setSecureItem,
    getSecureItem,
    removeSecureItem,
}
export default storage
```